soy/elements.soy

```diff
@@ -124,6 +124,7 @@
     <span class="firebaseui-idp-icon-wrapper">
       <img
           class="firebaseui-idp-icon"
+          alt="logo_{$providerId}"
```

Comment:
We may want to be consistent and do this for all images.
The alt text should be a readable text when the image can't be displayed. So something like  "{$providerId} logo" makes more sense.
Also this text could be internationalized (we now support that}, so you need to put it in a {msg}{/msg}, something like:
alt="{msg desc="The logo for a identity provider button."}{$providerId} logo{/msg}"